services/QuillLMS/config/application.rb

```diff
@@ -35,6 +35,7 @@ class Application < Rails::Application
       #{config.root}/app/services
       #{config.root}/app/services/analytics
       #{config.root}/app/queries/scorebook
+      #{config.root}/app/services/vitally_integration
```

Comment:
Were you getting errors without this line?

Reply:
Yeah, it has to be loaded here because otherwise this namespace doesn't get autoloaded.